helmfile.d/config/kusama/otv-backend.yaml.gotmpl

```diff
@@ -1464,7 +1464,7 @@ config: |
           },
           {
               "name": "amj_stake-0",
```

Comment:
If I understood Jonas correctly he also changed the telemetry name to "JUST JONAS/1" on both networks. Please update and merge records.